src/components/code_snippets/_bitbucket-semgrep-app-standalone.mdx

```diff
@@ -0,0 +1,19 @@
+  ```yaml
+  image: atlassian/default-image:latest
+  
+  pipelines:
+    default:
+      - parallel:
+        - step:
+            name: 'Run Semgrep scan with current branch'
+            deployment: dev
+            image: returntocorp/semgrep
+            script:
+              - export SEMGREP_RULES = "p/default"
+
+              # Uncomment the following line to scan changed 
+              # files in PRs or MRs (diff-aware scanning): 
+              # - export SEMGREP_BASELINE_REF = "main"
+
+              - semgrep ci
+  ```
```

Comment:
```suggestion
```yaml
image: atlassian/default-image:latest

pipelines:
  default:
    - parallel:
      - step:
          name: 'Run Semgrep scan with current branch'
          deployment: dev
          image: returntocorp/semgrep
          script:
            - export SEMGREP_RULES = "p/default"

            # Uncomment the following line to scan changed 
            # files in PRs or MRs (diff-aware scanning): 
            # - export SEMGREP_BASELINE_REF = "main"

            - semgrep ci
```
```

Reply:
I'll take a look @adamkvitek thanks. Yeah, the reason why I moved them out is because it's very possible we'll one day have to reorganize the docs so better for us to modularize it now rather than later (that is what I meant by futureproofing.)

Reply:
If you are already planning future documents using these code snippets, good idea! You included these code snippets in their own file, which is great... maybe we can even a subfolder for them - ci-configurations or something similar... but this is all option. Check the code indentation in the _bitbucket-semgrep-app-standalone.mdx also please. Thanks!